roles/pulp3-devel/files/settings.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
without looking at the pulp-smash settings area, I would guess that these are pulp settings. Please either rename or add a comment.